src/routes/(site)/blog/[id]/+page.svelte

```diff
@@ -6,7 +6,7 @@
   import Labels from '$lib/components/blog/labels.svelte';
   import BlogBody from './blog-body.svelte';
   import { onMount } from 'svelte';
```

Comment:
In the onMount function of this file, resolve any lint errors.
<img width="421" alt="image" src="https://user-images.githubusercontent.com/31261035/197375004-8fc4bf2b-a755-4a98-b7ee-cf2de58b4e7c.png">

Reply:
done